compiler/AST/build.cpp

```diff
@@ -376,6 +377,15 @@ static void addModuleToSearchList(UseStmt* newUse, BaseAST* module) {
   }
 }
 
+static void addModuleToSearchList(ImportStmt* newImport, BaseAST* module) {
+  UnresolvedSymExpr* modNameExpr = toUnresolvedSymExpr(module);
+  if (modNameExpr) {
+    addModuleToParseList(modNameExpr->unresolved, newImport);
+  } /* else if (CallExpr* callExpr = toCallExpr(module)) {
+    addModuleToSearchList(newUse, callExpr->argList.first());
+    }*/ // Expected to be useful when import statements can list multiple mods
```

Comment:
Not sure it's worth keeping this commented-out code. If you do keep it, put the comment about when it would be useful first, please. Another reasonable approach is to leave that code in (it won't be called, right?).

Reply:
I just merged it with the UseStmt version into a single function that handles their parent type instead.